### PR TITLE
#439: Add custom role definition

### PR DESCRIPTION
**Pull request by deveth0**
This Pull Request adds documentation on custom roles (see #439 for more information)

### CODE REVIEW

File: docs/src/main/asciidoc/security-service-accounts.adoc [hunk 1]
```diff
@@ -20,4 +20,56 @@ For earlier versions, it needs to be specified as an environment variable to the
...
+|pods, services, endpoints
+
+|spring-cloud-starter-kubernetes-config
+|configmaps, secrets
+
+|spring-cloud-starter-kubernetes-ribbon
+|pods, services, endpoints
+|===
+
+For development purposes, you can add `cluster-reader` permissions to your `default` service account. On a productive system you'll want to provide more granular permissions.
```
Comment (geoand) [inline docs/src/main/asciidoc/security-service-accounts.adoc]: `productive` -> `production`

Comment (geoand) [reply docs/src/main/asciidoc/security-service-accounts.adoc]: `you'll want` would probably better as `you'll likely want`

Comment (deveth0) [reply docs/src/main/asciidoc/security-service-accounts.adoc]: +1 thanks!

Comment (geoand) [reply docs/src/main/asciidoc/security-service-accounts.adoc]: YW!

**Pull request merged**